request.go

```diff
@@ -124,6 +146,19 @@ func (c *Client) delete(ctx context.Context, path string) error {
 	return nil
 }
 
+func (c *Client) action(ctx context.Context, path string, resp interface{}) error {
```

Comment:
Why don't name it post(...) if doing post request?

Reply:
Because this method expects that we will get `http.StatusOK` on success, like many our actions did. Post method is more generic stuff from my point of view.